.hlint.yaml

```diff
@@ -53,7 +56,7 @@
 - warn: {lhs: Data.Text.Lazy.readFile, rhs: Data.Text.Extended.readFileUtf8}
 - warn: {lhs: Data.Text.Lazy.writeFile, rhs: Data.Text.Extended.writeFileUtf8}
 - warn: {lhs: System.Environment.setEnv, rhs: System.Environment.Blank.setEnv}
-- warn: {lhs: toNormalizedFilePath, rhs: toNormalizedFilePath'}
+- warn: {lhs: toNormalizedFilePath, rhs: "toNormalizedFilePath'"}
```

Comment:
Did this not work? I thought I tested this.

Reply:
It messes up the yaml syntax highlighting in my editor. My yaml is not the best, but this change doesn't make things worse, does it?

Reply:
Should be fine, I just got slightly confused.